definitions/variable/energy/energy-secondary.yaml

```diff
@@ -1,11 +1,18 @@
 # List of variables related to the secondary energy sectors
 
+- Secondary Energy:
+    description: Total secondary energy
```

Comment:
Please provide a clear description what is counted here. If I turn crude oil into diesel and then burn it in a powerplant, is this counted twice?

Reply:
Maybe I am not the best person to answer that, but I would assume that to be consistent you need to report only the net secondary energy use here so, for example, if you produce a secondary energy to be used in another secondary energy process, this should be discounted from the total sum. 
Isn't there already an standard for that variable from other projects and/or reportings?

Reply:
In the ENGAGE project, I found the following:

> total secondary energy - the sum of all secondary energy carrier production (for consistency checks)

Not sure if this is particularly helpful...

Reply:
I am fine with that alternative, as this is what we do currently in REMIND.

Reply:
My understanding until now is that yes, total SE is the sum of all SE productions - so if you convert electricity to H2 to el to H2, you continually increase the SE amounts. 
This is one of the reasons why SE values can be (much) larger than FE values. 
I think if we would switch to a net accounting of SE, we would loose tractability and information - we already have the FE reporting to take note of what finally arrives at an end user.
But yes, I agree, the original PE/SE/FE split was much nicer before there were SE2SE conversions